cl/users/templates/profile/settings.html

```diff
@@ -54,6 +54,17 @@
                         {% endfor %}
                     </p>
                 {% endif %}
+                {% if EMAIL_BROKEN_BANNER.show %}
+                  {% if EMAIL_BROKEN_BANNER.type == "PERMANENT" %}
+                    <div class="alert alert-danger" role="alert">
+                      {{ email_error_message }}
+                    </div>
+                  {% else %}
+                    <div class="alert alert-warning" role="alert">
+                      {{ email_error_message }}
+                    </div>
+                  {% endif %}
```

Comment:
This is another good opportunity to combine duplicate code.
 
and/or remove this from the html.  We have the ability to pass messages in Django that can be used.  We dont need to pass email_error_message into the page as its own special variable.

Reply:
Replaced this code with a new simpler one since we are now using the default Django messages feature, instead of displaying the error message underneath the email address, now the email address field is only wrapped up in an alert box if there is some problem and display the error message above.